HedgeModManager/Resources/Codesv2/SonicFrontiers.hmm

```diff
@@ -8900,7 +8900,7 @@ Patch "Force Xbox One Controller UI" in "UI" by "Hyper"
     );
 }
 
-Patch "Force Xbox Series Controller UI" in "UI" by "Hyper"
+Patch "Force Xbox Series S/X Controller UI" in "UI" by "Hyper" does "Forces the Xbox Series S/X controller UI to be used."
```

Comment:
Change "S/X" to "X|S", as that's how it is branded. The description is redundant.